composer.json

```diff
@@ -16,7 +16,7 @@
         "behat/mink" : "dev-master#9ea1cebe",
         "drush/drush": "^9.0",
         "phpunit/phpunit": "^6.5",
-        "webflo/drupal-core-require-dev": "^8.7.0"
+        "webflo/drupal-core-require-dev": "^8.7"
```

Comment:
Package webflo/drupal-core-strict is abandoned, you should avoid using it. Use drupal/core-recommended instead.

This might apply to webflo/drupal-core-require-dev too. Maybe we should check this.

Reply:
@minnur  I changed `webflo/drupal-core-require-dev` for `drupal/core-dev`. Please note that this is a `require-dev` dependency within the module, which mean it's not installed when someone download the module - I believe we only keep it there for travis tests, which runs `wikimedia/composer-merge-plugin` and uses those dev requirements on the test environment.

Reply:
> Looks like drupal-composer project is using [drupal/core-composer-scaffold](https://github.com/drupal/core-composer-scaffold) for this, which is what we patterned our composer project after.

@cnovak Isn't that for the Kickstart project template? I don't see that applying to the `apigee_edge` module.